core/cloudflow-akka-testkit/src/main/scala/cloudflow/akkastream/testkit/TestContext.scala

```diff
@@ -63,7 +64,7 @@ private[testkit] case class TestContext(
       )
       .getOrElse(throw TestContextException(inlet.name, s"Bad test context, could not find source for inlet ${inlet.name}"))
 
-  def flowWithOffsetContext[T](outlet: CodecOutlet[T]): cloudflow.akkastream.scaladsl.FlowWithOffsetContext[T, T] = {
+  def flowWithCommittableContext[T](outlet: CodecOutlet[T]): cloudflow.akkastream.scaladsl.FlowWithCommittableContext[T, T] = {
```

Comment:
Are we not deprecating this method like the others?

Reply:
I think this method should have been private.

Reply:
(I'll make it private)